node_modules/oae-preview-processor/lib/api.js

```diff
@@ -507,11 +507,13 @@ var _generatePreviews = function(ctx, callback) {
         if (err) {
             return callback(err);
         } else if (processor) {
-            processor.generatePreviews(ctx, ctx.content, function(err, ignored) {
+            processor.generatePreviews(ctx, ctx.content, function(err, ignored, pending) {
                 if (err) {
                     ctx.setStatus('error', callback);
                 } else if (ignored) {
                     ctx.setStatus('ignored', callback);
+                } else if (pending) {
+                    ctx.setStatus('pending', callback);
                 } else {
```

Comment:
Actually it may already be in `node_modules/oae-content/lib/constants.js` but not being used here. Can you update this to use them?

Note that test references don't have to be refactored for it.